CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
+<a name="1.3.6"></a>
+
+## [1.3.6](https://github.com/openmail/system1-cmp/compare/v1.3.4...v1.3.6) (2020-02-20)
+
+### Fix
+
+- [x] Add postMessage call to init
```

Comment:
can you note in this that the change only affects the info-acs reference page? im in process of splitting out info-acs now